### ISSUE TITLE
Truncate link not visible to screenreaders

### ISSUE THREAD

**Issue by samzhang111** [One-com/react-truncate#81]
Allowing a tab index to be set on the link would solve this issue. Even better, would be allowing in `linkProps` that are passed directly to the link.

**Comment by samzhang111** [comment]
Nvm I misunderstood how this worked
